profile/build_test.go

```diff
@@ -40,8 +41,26 @@ func (b BuildTest) SetupGomega(t *testing.T) BuildTest {
 
 func (b BuildTest) SetupWorkspace() BuildTest {
 	var err error
+	b.context.Buildpack.Path = "../scripts"
```

Comment:
```suggestion
	b.context.Buildpack.Path = "../"
```

Also necessary if you make the other changes I've suggested.